lib/sql.js

```diff
@@ -540,6 +548,53 @@ SQLConnector.prototype.buildInsert = function(model, data, options) {
   return this.parameterize(insertStmt);
 };
 
+/**
+ * Build INSERT SQL statement for multiple values
+ * @param {String} model The model name
+ * @param {Object} data The array of model data object
+ * @param {Object} options The options object
+ * @returns {Object} The ParameterizedSQL Object with INSERT SQL statement
+ */
+SQLConnector.prototype.buildInsertAll = function(model, data, options) {
+  if (!this.multiInsertSupported) {
+    debug('multiple value insert SQL dialect is not supported by this connector');
```

Comment:
i wonder if we want to throw an error in this case.

Reply:
Actually, this flag will be checked at juggler level and rejected from there itself. I'll be raising that PR soon. I have added this to ensure backwards compatibility in connector code too. Let me know if you still feel we should throw error.